mount/mount_windows.go

```diff
@@ -132,14 +132,6 @@ func (m *Mount) GetParentPaths() ([]string, error) {
 
 // Unmount the mount at the provided path
 func Unmount(mount string, flags int) error {
-	if _, err := os.Stat(mount); err != nil {
-		if os.IsNotExist(err) {
-			return nil
-		}
-
-		return fmt.Errorf("failed to access mount point %s: %w", mount, err)
-	}
-
```

Comment:
Why was this block removed? AFAIR it's necessary because `Unmount("/nosuchpath", 0)` is valid and supposed to be a no-op. Or are we just relying on `bindfilter.RemoveFileBinding` to implement that?

Reply:
I added it previously in https://github.com/containerd/containerd/pull/8043/commits/6a5b4c9c242a0278033b20f7f5eca4fdbbea231a. It felt wrong to mask an error that the target is not there. On Linux, this function ends up calling ```unix.Unmount``` which will err out if the target does not exist.

Should I leave it in?

Reply:
I think we crossed edits. Linux `Unmount` doesn't ignore this case, but `UnmountAll` does. My initial comment was confused with `""` handling which we already have in Windows `UnmountAll`.

Reply:
in that case I think we can move that check in ```UnmountAll()```. The bind filter will probably not return an error that matches ```IsNotExist```.

Reply:
Ok. Same thing on linux: https://github.com/containerd/containerd/blob/main/mount/mount_linux.go#L181-L183

Reply:
added https://github.com/containerd/containerd/pull/8043/commits/c9e5c33a18acffe583592828a5a6696cbc779c1a